test/node/parseIDLFile-helper.js

```diff
@@ -0,0 +1,87 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+global.parseIDLFileTest = function(data) {
+  describe(data.description, function() {
+    var execSync = require('child_process').execSync;
+    var originalTimeout;
+    var LocalGitRunner;
+    var outputBox;
+    var errorBox;
+
+    afterAll(function() {
+      execSync(`/bin/rm -rf "${data.localRepositoryPath}"`);
+    });
+
+    beforeEach(function() {
+      foam.CLASS({
+        package: 'org.chromium.webidl.test',
+        name: 'AccumulatorBox',
+        implements: [ 'foam.box.Box' ],
+
+        properties: [
+          {
+            class: 'Array',
+            name: 'results',
+          },
+        ],
+
+        methods: [
+          function send(message) {
+            this.results.push(message.object);
+          },
+        ],
+      });
+
+      LocalGitRunner = foam.lookup('org.chromium.webidl.LocalGitRunner');
+      var AccumulatorBox = foam.lookup('org.chromium.webidl.test.AccumulatorBox');
+      outputBox = AccumulatorBox.create();
+      errorBox = AccumulatorBox.create();
+
+      data.GitilesIDLFile = foam.lookup('org.chromium.webidl.GitilesIDLFile');
+      data.GithubIDLFile = foam.lookup('org.chromium.webidl.GithubIDLFile');
+      data.Parser = foam.lookup(`org.chromium.webidl.${data.parser}`);
+      data.IDLFileContents = foam.lookup('org.chromium.webidl.IDLFileContents');
+
+      originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+      jasmine.DEFAULT_TIMEOUT_INTERVAL = 10000000;
+
+      // Must not be in place before each test (including first).
+      execSync(`/bin/rm -rf ${data.localRepositoryPath}`);
```

Comment:
I also like to live dangerously.

Put quotes around the path argument, because you *really* don't want spaces in paths interpreted by command-line parsing in this case.

I'd also recommend reworking this to use a generated path in /tmp (man mktemp) instead of a hardcoded path inside the workspace. Dumping temporary test data into a workspace is rarely a good idea.